ngraph/frontend/onnx_import/src/op/dequantize_linear.cpp

```diff
@@ -128,8 +128,15 @@ namespace ngraph
                                                                 const int64_t axis,
                                                                 const PartialShape& x_shape)
                     {
-                        std::vector<int64_t> target_dims;
+                        auto input_rank = input.get_partial_shape().rank();
 
+                        // Do not reshape input, if it contains a scalar value
+                        if (input_rank.is_static() && input_rank.get_length() == 0)
+                        {
+                            return input.get_node_shared_ptr();
```

Comment:
Can we add tests to cover the new behavior?

Reply:
Sure, done